### PR TITLE
fix(metrics): near_client_tracked_shards metric tracks all tracked shards

### PR DESCRIPTION
**Pull request by nikurt**

### CODE REVIEW

**Comment by akhi3030** [comment]
@posvyatokum: may I ask you to review please?

File: chain/client/src/info.rs [hunk 1]
```diff
@@ -135,13 +135,12 @@ impl InfoHelper {
...
         if let Ok(num_shards) = client.epoch_manager.num_shards(&head.epoch_id) {
             for shard_id in 0..num_shards {
-                let tracked = match me {
-                    None => false,
-                    Some(me) => client
-                        .epoch_manager
-                        .cares_about_shard_from_prev_block(&head.last_block_hash, me, shard_id)
-                        .unwrap_or(false),
-                };
+                let tracked = client.shard_tracker.care_about_shard(
```
Comment (posvyatokum) [inline chain/client/src/info.rs:138]: Just for my own education. 
`care_about_shard` is what we are using to actually track shards, right? This is THE place where we smoosh together tracked_config and EpochInfo, and decide what to do if node is a validator with some unusual configuration?

**Review by posvyatokum — APPROVED** [review]
LG(reat)TM if I guessed correctly in my question

File: chain/client/src/info.rs (hunk 1 above)
Comment (nikurt) [reply chain/client/src/info.rs:138]: Yes, right.
`EpochManager::cares_about_shard_from_prev_block()` doesn't take into account `tracked_shards` and `tracked_accounts`.

**Pull request merged**